test/fixtures/pre_release_package_info_response.dart

```diff
@@ -0,0 +1,428 @@
+// ignore_for_file: prefer_single_quotes, lines_longer_than_80_chars
+// Generated from: https://pub.dev/api/packages/very_good_cli
+
+import 'dart:convert';
+
+final preReleasePackageInfoResponseBody = json.encode(
+  preReleasePackageInfoResponse,
+);
+
+const preReleasePackageInfoResponse = {
```

Comment:
Why is this file a dart file and not just the actual json from `// Generated from: https://pub.dev/api/packages/mason`?

Reply:
Maybe @felangel has more context here?

Reply:
yep, anyway, not changing it in this pr since it is causing no harm to testing purposes.